### ISSUE TITLE
Appium helper doesn't understand host configuration 

### ISSUE THREAD

**Issue by trinhpham** [Codeception/CodeceptJS#1724]
#### What are you trying to achieve?
I want to specify Appium server on other host

#### What do you get instead?
CodeceptJS keeps working with localhost

> Provide console output if related. Use `--verbose` mode for more details.

```bash
    Emitted | test.before ([object Object])
2019-06-18T06:48:49.215Z ERROR webdriver: Request failed due to Error: connect ECONNREFUSED 127.0.0.1:4723
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1097:14)
    [1] Error | Error: Can't connect to WebDriver.
Error: connect ECONNREFUSED 127.0.0.1:4723
```

### Details

 ```bash
npm --versions
{ codeceptjs: '2.1.4',
  npm: '6.4.1',
  ares: '1.15.0',
  cldr: '35.1',
  http_parser: '2.8.0',
  icu: '64.2',
  modules: '64',
  napi: '3',
  nghttp2: '1.34.0',
  node: '10.15.3',
  openssl: '1.1.0j',
  tz: '2019a',
  unicode: '12.1',
  uv: '1.23.2',
  v8: '6.8.275.32-node.51',
  zlib: '1.2.11' }
```
* WebDriverIO: 5.10.4
* Appium: 1.13.0
* Configuration file:

```js
{
    "helpers": {
      "Appium": {
        "host": "192.168.160.53",
        "port": 4444,
        "platform": "android",
        "desiredCapabilities": {
          "appium:app": "http://url.to/my.apk",
          "appium:deviceName": "sample"
        }
      }
   }
}
```

I've made a PR for this at #1723

**Comment by trinhpham** [comment]
PS: If anyone curious about the reason of having prefix `appium:` in the desired capabilities, you should know that these caps are not in standard of [WebDriver capabilities](https://www.w3.org/TR/webdriver1/#capabilities) so the Selenium Hub cannot understand. Adding that prefix will help the hub ignore these caps then forward them to the Appium Node

**Comment by DavertMik** [comment]
I think this was fixed in 2.1.5
